### ISSUE TITLE
Get rid of "Media saved by" receipt

### ISSUE THREAD

**Issue by Sepero** [session-foundation/session-android#550]
A recent addition to the latest version of Session is notification of Media saved by the receiver in 1 to 1 chats.

In my humble opinion, the app shouldn't be sending this information, but please let me explain reasons why.

1. If I send a message, then that was purposeful/intentional.
2. If I send a file, then that was intentional.
3. If I send a link with a preview, that I send is intentional, and under my control.

When I save a image or file, I do not intend to send data across the network. Letting others know that the file was saved is not essential information to our communications. In secure communications, I should think it undesirable to unintentionally/involuntarily be sending nonessential information across the network.

Also, if someone wanted to, they could modify the source code, and delete the code sending information. They could save all media, and the sender wouldn't know. Therefore, if a image or file is sent, we should assume by default that it was saved.

TLDR; 1) Communicating this information is not essential. 2) Secure communication should be intentional. 3) Ability for anyone to modify and compile their own client code means we can never trust whether the message is even true or not

My request: In an effort to keep Session a Simple app, I respectfully request that this new feature be removed. If not removed, made into a toggle option in Settings.

Lastly, please don't take offense to this issue. I understand that this is a new feature, and it takes pain staking time to implement such things. We all greatly appreciate all the work and development that goes into Session. Session is loved and very much needed.

PS.
I think this issue highlights a broader issue regarding security, and setting standards for what information should and should not be transmitted. The set of fundamental Standards/Guidelines for Session development.

**Comment by vanontom** [comment]
Agreed with Sepero. I don't really understand the purpose of the notification. It seems unnecessary, and honestly kinda creeps me out every time (like I'm invading *their* privacy). I have "name only" notifies set, so I have to fully open app just to see it was only a "Media saved by..." event.

Suggestions:
Setting to disable (by default, more like other apps).
Notification category (possible to disable or silence).
